closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -1155,7 +1242,10 @@ public synchronized int doFinal(byte[] in, int inOff, int inLen, byte[] out,
 
         @Override
         public int getCipherBufferLength() {
-            return cipherBuf.size();
+            if (cipherBuf != null) {
+                return cipherBuf.size();
+            }
+            return 0;
```

Comment:
Suggest using the ternary operator here:
```java
            return (cipherBuf != null) ? cipherBuf.size() : 0;
```

Reply:
This code matches what is in the non native ChaCha20 class. Id prefer to keep it such that the classes can be more easily diffed but made this change anyway. If its ok id like to change it back to match the non native class for maintenance reasons.